src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -116,6 +117,23 @@ public AddCommand parse(String args) throws ParseException {
             Skill skill = new Skill(name, level, tagList);
             return new AddSkillCommand(skill);
 
+        case(ItemUtil.NOTE_ALIAS):
+            if (!argMultimap.getPreamble().isEmpty()) {
+                throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                        AddNoteCommand.MESSAGE_USAGE));
+            }
+
+            if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_TIME)) {
+                throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                        AddNoteCommand.MESSAGE_USAGE));
+            }
+
+            name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
+            Time noteTime = ParserUtil.parseTime(argMultimap.getValue(PREFIX_TIME).get());
+            tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+
+            Note note = new Note(name, noteTime, false, tagList);
```

Comment:
If isDone of note is always false when instantiated, can we just remove this and put it in the constructor?